app/models/host.rb

```diff
@@ -610,7 +610,7 @@ def is_scannable_now_error_message
   end
 
   def is_vmware?
-    vmm_vendor.to_s.strip.downcase == 'vmware'
+    vmm_vendor_display.to_s.strip.downcase == 'vmware'
```

Comment:
I feel like this will make a lot easier to get rid of these terrible conditionals. :+1: